web/src/main/ui/app/components/flows-new/edit-flow/ingest/ingest.component.ts

```diff
@@ -37,7 +47,6 @@ export class IngestComponent implements OnInit {
     } = this.step.fileLocations;
 
     const {
-      additionalCollections,
```

Comment:
@brucean52 was this part of DHFPROD-2561? Want to make sure working code isn't deleted with this PR.